core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackService.java

```diff
@@ -393,6 +393,10 @@ public StackV4Response getByAmbariAddress(String ambariAddress) {
         return stackRepository.findByEnvironmentCrnAndStackType(environmentCrn, stackType);
     }
 
+    public List<Stack> getStackByEnvironmentCrnAndStackType(String environmentCrn, StackType stackType, Long workspaceId) {
```

Comment:
I checked out the code and this method is used nowhere. Can we delete it?

Reply:
Deleted.